### ISSUE TITLE
Optional Arguments

### ISSUE THREAD

**Issue by darrencurtis** [awslabs/aws-instance-scheduler#18]
Hi,
Thanks for the awesome tool.  I need to schedule very specific periods, day, month, & times, however the  "scheduler-cli create-period" does NOT support --months, like the documentation states:  https://docs.aws.amazon.com/solutions/latest/instance-scheduler/components.html

I receive the following error:  scheduler-cli: error: unrecognized arguments: --months 4

Performing a "help" -h on the command indeed does not list --months. 

Will this option become available, as it would be most helpful?

optional arguments:
  -h, --help            show this help message and exit
  --begintime BEGINTIME
                        Begin time of the period in hotmat hh:mm
  --description DESCRIPTION
                        Description for the period
  --endtime ENDTIME     End time of the period in format hh:mm
  --monthdays MONTHDAYS
                        Calendar monthdays of the period
  --name NAME           Name of the period
  --weekdays WEEKDAYS   Weekdays of the period
  --query QUERY         JMESPath query to transform or filter the result
  --region REGION       Region in which the Instance Scheduler stack is
                        deployed
  --stack STACK, -s STACK
                        Name of the Instance Scheduler stack

Thanks
Darren

**Comment by arieleeuw** [comment]
Hi Darren,
I'll put these on out  list for a next updatr, In the meantime if you want to add them yourself by just inserting 3 lines of code to pass the parameter to the cli handler of the scheduler that runs as part of the Instance Scheduler Lambda function.

1. Download the scheduler-cli.zip and extract it
2. Open the file scheduler_cli.py
3. Locate line HELP_PERIOD_MONTH_DAYS = "Calendar monthdays of the period" and insert line 
**HELP_PERIOD_MONTHS = "Months"** of the period" below it

```
HELP_PERIOD_MONTH_DAYS = "Calendar monthdays of the period"
HELP_PERIOD_MONTHS = "Months of the period"
```
4. Locate PARAM_MONTHDAYS = "--monthdays" and insert the line **PARAM_MONTHS = "--months"**  below it
```
PARAM_MONTHDAYS = "--monthdays"
PARAM_MONTHS = "--months"
```
5. Locate period_parser.add_argument(PARAM_MONTHDAYS, help=HELP_PERIOD_MONTH_DAYS) and insert **period_parser.add_argument(PARAM_MONTHS,help=HELP_PERIOD_MONTHS)** below it

```
    period_parser.add_argument(PARAM_MONTHDAYS, help=HELP_PERIOD_MONTH_DAYS)
    period_parser.add_argument(PARAM_MONTHS,help=HELP_PERIOD_MONTHS)
```

Please let me know if you have any issues

Arie

**Comment by darrencurtis** [comment]
I can confirm that worked well, except I needed to add a space in the number 5 insert.  Posted here just in case anyone else just blindly copies and pastes :-)
period_parser.add_argument(PARAM_MONTHS, help=HELP_PERIOD_MONTHS)
                                                  Added a space  ^
Cheers,
Darren

**Comment by shsenior** [comment]
Fixed in v1.2.0
